tsl/src/continuous_aggs/invalidation.c

```diff
@@ -196,7 +192,7 @@ invalidation_cagg_log_add_entry(int32 cagg_hyper_id, int64 modtime, int64 start,
 	HeapTuple tuple;
 
 	Assert(start <= end);
-	tuple = create_invalidation_tup(RelationGetDescr(rel), cagg_hyper_id, modtime, start, end);
+	tuple = create_invalidation_tup(RelationGetDescr(rel), cagg_hyper_id, start, end);
```

Comment:
You are removing the use of `modtime` here, which leaves an unused input parameter/argument. I am sure this also leaves a bunch of now unnecessary code to compute the modification time that is no longer needed (basically dead code).

Reply:
Most of the modification time parameter was taken from the invalidation records, which are removed. There were essentially no computations done for the modification parameter, it was just filled in in one place and then propagated, so I see no more code to remove. If you have any suggestions, please point me to where that would be.

Reply:
This function (`invalidation_cagg_log_add_entry`) still takes `modtime`, which I guess is one reason you haven't seen more code to remove. The `modtime` parameter, although never used, is still passed into the function here. 

in other words, you only removed `modtime` in the function body, but not in the parameter list that would affect the signature of the function.

Reply:
Yes, that was it. Surprised that I didn't get an error inside the function though, since the parameter was not used.